cli-feed-v4.json

```diff
@@ -9640,7 +9640,8 @@
               "hidden": false,
               "displayVersion": "v4",
               "targetFramework": ".NET",
-              "description": "LTS"
+              "description": "LTS",
+              "endOfLifeDate":"2024-11-12T00:00:00.0000000Z"
```

Comment:
@cloudmelon Could you double check the dates?

Reply:
it is when Windows is supported. I think we have windows 10 and windows 11